### ISSUE TITLE
Home page

### ISSUE THREAD

**Issue by himanshu007-creator** [himanshu007-creator/SFC-foundations#5]

**Comment by RuchiGitHu** [comment]
Good Evening,
I am looking forward to solve this issue...
Please let me know something about this issue for my clarity..

**Comment by himanshu007-creator** [comment]
hi @RuchiGitHu , so the issue is to create a new home page. We are building the website right from scratch and would love to redesign everything. That's why, we will be creating a new HOME PAGE as well. I hope it gives a basic idea.

**Comment by Anubhav-developr** [comment]
Ok sir i visited the website ...
It was pretty awesome 
I think sir you want homepage in the hamburger menu is it so ?

**Comment by himanshu007-creator** [comment]
> Ok sir i visited the website ...
> It was pretty awesome
> I think sir you want homepage to get accessed from the hamburger menu is it so ?

sort of , Yeah , but not exactly

**Comment by Anubhav-developr** [comment]
Ok sir .... btw your website was pretty awesome sir 👌

**Comment by himanshu007-creator** [comment]
> Ok sir .... btw your website was pretty awesome sir 👌

Thanks but, it was because of WORDPRESS TEMPLATES.

**Comment by RuchiGitHu** [comment]
> hi @RuchiGitHu , so the issue is to create a new home page. We are building the website right from scratch and would love to redesign everything. That's why, we will be creating a new HOME PAGE as well. I hope it gives a basic idea.

Thanks for your help...
Now, I understood what I have to contribute for this issue...

**Comment by ayushraghuwanshi7500** [comment]
Hi, all what are the tech stacks we can use? can we use bootstrap and may be react?

**Comment by himanshu007-creator** [comment]
> Hi, all what are the tech stacks we can use? can we use bootstrap and may be react?

Yes, you can but keep in mind, we have to make it simple, because ultimately non-technical person will be the admin and using this website. @ayushraghuwanshi7500  You can work on  #3 that will be more helpful. I want a system that enables admin to login and upload blogs with photographs.  <a href="http://sfcfoundations.com/category/blogs/">HERE</a> you can get an idea what feature i am asking you for.
I Hope you will work on this

**Comment by himanshu007-creator** [comment]
Have a look here first of all https://github.com/himanshu007-creator/SFC-foundations/issues/12

**Comment by sneha-nitdgp** [comment]
I'd like to work on the login feature to upload blogs along with photographs. Can I use django for the same?

**Comment by kunal299** [comment]
@himanshu007-creator I would love to contribute to the login feature as a part of Gssoc'21?

**Comment by nisha331** [comment]
hey @himanshu007-creator i want to work on the front-end of this project as a part of GSSOC'21 , when and from where should i start

**Comment by medhavi11** [comment]
I would like to work on this issue, I am GSSOC 2021 participant.

**Comment by himanshu007-creator** [comment]
@nisha331   yes you can work on it. We just need an idea like how you will proceed. As we are redesigning everything so everything will be changes and so will the site structure. We just need to know how you are going to implement it. Prepare a prototype of code and send it within this week. Best of luck!

**Comment by bug-debug-done** [comment]
> @nisha331 yes you can work on it. We just need an idea like how you will proceed. As we are redesigning everything so everything will be changes and so will the site structure. We just need to know how you are going to implement it. Prepare a prototype of code and send it within this week. Best of luck!

can I also creat a prototype?

**Comment by machine-sg** [comment]
I am a GSSOC'21 participant and I wish to work on this. Hope to hear back from you . 
Thank you .

**Comment by mansijain980** [comment]
@himanshu007-creator can i work on this issue ..... i m thinking of using containers and some attraction google fonts with responsive pages. 
Please assign this to me.

**Comment by Aastha1011** [comment]
Hi I am Aastha Agarwal, a GSSOC'21 participant. Can you assign me this issue!? I will be using HTML, CSS, BootStrap and JavaScript to design it.

**Comment by Dimple115** [comment]
Hey, I'm gssoc'21 participant, I would like to give my hand in this project. Please assign this issue to me.

**Comment by bindu030** [comment]
Hi I am Bindu GSSOC'21 participant would love to design a new web page please assign this task to me.Thank you

**Comment by mansijain980** [comment]
> @himanshu007-creator can i work on this issue ..... i m thinking of using containers and some attraction google fonts with responsive pages.
> Please assign this to me.

Let me know sir . i m excited to work on it ...please assign it to me.

**Comment by rohank2502** [comment]
We are having Home Page can we close this issue ? @himanshu007-creator

**Comment by himanshu007-creator** [comment]
I wanted her page suggestion saved for an alternative someday, but she is inactive,

**Comment by ria-19** [comment]
Is this still open? I would like to contribute to this project.
